ostd/src/sync/rcu/mod.rs

```diff
@@ -4,28 +4,26 @@
 
 use core::{
     marker::PhantomData,
-    ops::Deref,
     ptr::NonNull,
     sync::atomic::{
         AtomicPtr,
         Ordering::{AcqRel, Acquire},
     },
 };
 
+use non_null::NonNullPtr;
 use spin::once::Once;
 
 use self::monitor::RcuMonitor;
 use crate::task::{atomic_mode::AsAtomicModeGuard, disable_preempt, DisabledPreemptGuard};
 
 mod monitor;
-mod owner_ptr;
-
-pub use owner_ptr::OwnerPtr;
+pub mod non_null;
```

Comment:
Actually, `non_null::NonNullPtr` seems unnecessarily verbose. I'm not sure why you made these changes.

Reply:
> This is mainly used for non_null::ArcRef/BoxRef....

Thanks for the explanation! I forgot about the `Ref` types when checking the changes. Then the changes seem reasonable.

> Should we need to re-export NonNullPtr, which will make users have two paths to get NonNullPtr?

It will cause consistency problems... The current form should be reasonable enough.